pydantic/_internal/_docs_extraction.py

```diff
@@ -0,0 +1,62 @@
+"""Utilities related to attribute docstring extraction."""
+from __future__ import annotations
+
+import ast
+import inspect
+import textwrap
+from typing import Any, Iterable
+
+
+def _get_assign_targets(node: ast.Assign | ast.AnnAssign) -> Iterable[str]:
+    if isinstance(node, ast.Assign):
+        for target in node.targets:
+            if isinstance(target, ast.Tuple):
+                yield from (el.id for el in target.elts)
+            else:
+                yield target.id
+    else:
+        # `target` can only be `ast.Name` in the context of a `BaseModel` subclass definition
+        yield node.target.id
```

Comment:
What's the scenario that we need to supporrt `ast.Assign`? It's not possible to build a `BaseModel` with them, is it?